src/components/Cell.svelte

```diff
@@ -0,0 +1,55 @@
+<script>
+  import { getCellSize, getCellDefaultProps } from '../helpers/cell.js';
+  import { fly } from 'svelte/transition';
+  import { onMount } from 'svelte';
+
+  export let total = 1;
+  export let color = '#c1c1c1';
+  export let size = null;
+  export let additionalClassName = '';
+  export let popupText = '';
+
+  let padding = total < 500 ? 3 : 1;
```

Comment:
this could be `const`
It's a simple rule - use `const` by default and then change to `let` if needed